lib/internal/test_runner/utils.js

```diff
@@ -55,11 +55,10 @@ const kRegExpPattern = /^\/(.*)\/([a-z]*)$/;
 
 const kPatterns = ['test', 'test/**/*', 'test-*', '*[._-]test'];
 const kFileExtensions = ['js', 'mjs', 'cjs'];
-if (getOptionValue('--experimental-strip-types')) {
-  ArrayPrototypePush(kFileExtensions, 'ts', 'mts', 'cts');
-}
 const kDefaultPattern = `**/{${ArrayPrototypeJoin(kPatterns, ',')}}.{${ArrayPrototypeJoin(kFileExtensions, ',')}}`;
 
+const kDefaultTSPattern = '**/test/**/*{-,.,_}test.{cts,mts,ts}';
```

Comment:
Can we avoid having this second pattern that is passed around all over the place? Ideally we would just update `kDefaultPattern` to also match the desired TS files.